CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.3.0
+
+* Add Response.badRequest() constructor
 ## 1.2.1-dev
```

Comment:
Delete this line. And roll the changes together. We'll just release 1.3.0

Reply:
Okay, I _think_ I did what you were asking? Not 100% sure I followed that. You wanted me to delete the line that said 1.2.1-dev, yeah?

Reply:
Ohh that's why you said to change it from 1.2.1-dev to 1.3.0. I understand now. Guess I'm just unfamiliar with that way of managing it but it makes sense